.travis.yml

```diff
@@ -3,19 +3,21 @@ dist: trusty
 os:
 - osx
 - linux
+- windows
+
 env:
-- NODE_ENV=dev
+- NODE_ENV=dev 
 - NODE_ENV=prod
 language: node_js
 node_js:
-- 8
-cache:
-  yarn: true
-  directories:
-  - node_modules
-  - app/node_modules
-  - "$HOME/.electron"
-  - "$HOME/.cache"
+- 8 
+# cache:
```

Comment:
@shriram2301 does it work with cache enabled?

(If so would be nice to reenable this)

Reply:
It fails if cache is enabled in windows. It happens consistently so for now let's keep it disabled.